src/room/GroupCallView.tsx

```diff
@@ -272,7 +272,16 @@ export function GroupCallView({
       );
     }
   } else if (left) {
-    if (isPasswordlessUser || PosthogAnalytics.instance.isEnabled()) {
+    // The call ended view is shown for two reasons: prompting guests to create
+    // an account, and prompting users that have opted into analytics to provide
+    // feedback. We don't show a feedback prompt to widget users however (at
+    // least for now), because we don't yet have designs that would allow widget
+    // users to dismiss the feedback prompt and close the call window without
+    // submitting anything.
+    if (
+      isPasswordlessUser ||
+      (PosthogAnalytics.instance.isEnabled() && !isEmbedded)
```

Comment:
Is there ever the case that we have a passwordless user in embedded mode?
Or do we just never want to show the call ended view in embedded mode. In this case you can change the condition to just never be true if embedded?
```suggestion
      (isPasswordlessUser || PosthogAnalytics.instance.isEnabled()) && !isEmbedded
```

Reply:
There can be no passwordless users in embedded mode, so the condition is equivalent to what you've suggested